app/src/main/res/xml/pdp_fragment_scene.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MotionScene xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <ConstraintSet android:id="@+id/start">
+        <Constraint
+            android:id="@+id/iv_product"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/height_iV_price"
```

Comment:
Please, rename dimen to `iv_product_height`

Reply:
done